src/main/java/hudson/remoting/AsyncFutureImpl.java

```diff
@@ -87,7 +85,7 @@ public synchronized V get() throws InterruptedException, ExecutionException {
 
     @Override
     @CheckForNull
-    public synchronized V get(@Nonnegative long timeout, TimeUnit unit) throws InterruptedException, ExecutionException, TimeoutException {
+    public synchronized V get(/* @javax.annotation.Nonnegative */ long timeout, TimeUnit unit) throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
We should just remove it rather than comment it out.

Reply:
Feedback that would have also applied to jenkinsci/jenkins@5674de12353df95d1c5f0baaa699636ed47b7284. I have done as you suggested.